.github/workflows/deploy-site.yaml

```
@@ -51,12 +51,8 @@ jobs:
       - name: Setup gcloud
         uses: google-github-actions/setup-gcloud@v1
 
-      - name: Populate list in publicsuffix.org
-        run: |-
-          cp list/public_suffix_list.dat publicsuffix.org/list/
-          cp list/public_suffix_list.dat publicsuffix.org/list/effective_tld_names.dat
-
       - name: Deploy website
         run: |-
-          gsutil -m -q rsync -d -r -x ".git" publicsuffix.org/ gs://${{ secrets.BUCKET_NAME }}/
-          gsutil -m -q setmeta -h "Cache-Control:public, max-age=86400" gs://${{ secrets.BUCKET_NAME }}/**
+          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -d -r -x ".git|.dat$" publicsuffix.org/ gs://${{ secrets.BUCKET_NAME }}/
```

Since the exclude pattern is a regular expression, I suggest changing it to
```suggestion
          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -d -r -x '\.git|\.dat$' publicsuffix.org/ gs://${{ secrets.BUCKET_NAME }}/
```
Otherwise, the leading dot is a bit misleading, since it matches an arbitrary character.